### ISSUE TITLE
Memory leak in dynamic rendering

### ISSUE THREAD

**Issue by Nberezhnoy** [konvajs/vue-konva#53]
Hey. I ran into a memory leak issue when drawing a list. My application dynamically retrieves the shapes from the server and renders them using a v-for directive. After some time, the memory consumption is about 2 gigabytes and the browser crashes. When monitoring performance, you can see that there are a huge number of undeleted nodes and event subscriptions.

![Снимок экрана от 2019-05-09 18-00-02](https://user-images.githubusercontent.com/3696830/57464761-f5b4e280-7285-11e9-92a7-689c5121b6a7.png)
![Снимок экрана от 2019-05-09 18-01-25](https://user-images.githubusercontent.com/3696830/57464762-f5b4e280-7285-11e9-9743-65ac39ef8cdd.png)

I give a part of the code at which there is a leak

```	
<v-group>
	<v-rect v-for="plate in processedPlates" :key="plate.id" :config="plate.config"></v-rect>
</v-group>
```

**Comment by lavrton** [comment]
How many objects do you have?

**Comment by lavrton** [comment]
The number of objects changes during the time?

**Comment by Nberezhnoy** [comment]
@lavrton Thank you for the quick response. I update the array with objects about 10 times per second.
The size of the array, which is used in the directive v-for, on average does not exceed 5 elements.

**Comment by lavrton** [comment]
How do you update it? Can you make a small demo?

**Comment by lavrton** [comment]
I think I see the leak: https://codesandbox.io/s/5mw10ml51k

**Comment by Nberezhnoy** [comment]
I get the event via the websocket and assign the value to an array, which is subsequently rendered. The concept is almost 1 in 1 like you have in the demo  

> https://codesandbox.io/s/5mw10ml51k

**Comment by lavrton** [comment]
Fixed with the last release `vue-konva@2.0.6`. Try it.

**Comment by lavrton** [comment]
Thanks for the report!
